recipe/recipe.yaml

```diff
@@ -34,7 +35,8 @@ requirements:
     - pytest
     - typing_extensions
     - python ${{ python_min }}.*
-    - dagster ==1.10.6
+    - dagster == {{ version }}
+    - dagster-shared=={{ dagster_library_version }}
```

Comment:
We first have to add this library to conda though - this ist still missing

Reply:
awesome: https://github.com/search?q=repo%3Aconda-forge%2Fdagster-feedstock%20dagster-shared&type=code has already added it